docs/governance/introduction-to-decred-governance.md

```diff
@@ -8,7 +8,8 @@ Decred's governance is based on the principle of ticket-holder voting. The ultim
 
 ---
 
-Holders of DCR can time-lock their funds in exchange for [tickets](../proof-of-stake/overview.md). Tickets allow one to participate in Decred's governance in three ways, two on-chain and one off-chain.
+Holders of DCR can time-lock their funds in exchange for [tickets](../proof-of-stake/overview.md). The ticket price (the amount that must be time-locked) is adjusted dynamically every 144 blocks (~12 hrs), in an attempt to in order to keep PoS subsidy returns stable over time. Tickets allow one to participate in Decred's governance in three ways, two on-chain and one off-chain. 
```

Comment:
grammar: `in an attempt to in order to`.

Reply:
Good catch. Went with `in an attempt to`.